client/verta/verta/_repository/commit.py

```diff
@@ -123,6 +127,154 @@ def _raise_lookup_error(path):
         e = LookupError("Commit does not contain path \"{}\"".format(path))
         six.raise_from(e, None)
 
+    # TODO: consolidate this with similar method in `_ModelDBEntity`
```

Comment:
Could you at least point to the differences? It's hard to review code that was already reviewed and spot the differences.